### PR TITLE
title: add multiport check and dns check feature

### LINKED ISSUE #10: add multiport check and dns check feature
**What would you like to be added**:
add multiport check and dns check feature

**Why is this needed**:

### PR DESCRIPTION
**Pull request by GreatLazyMan**
#### What type of PR is this?
/kind feature

#### What does this PR do?
add multiport check and dns check feature

#### Which issue(s) does this PR fix?
Fixes #10
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/command/share/do.go [hunk 1]
```diff
@@ -112,35 +133,80 @@ func (o *DoOptions) RunRange(iPodInfos []*FloatInfo, jPodInfos []*FloatInfo) []*
...
-					if o.Skip(iPodInfo, targetIP) {
-						continue
+		var cmdObj command.Command
+		var target string
+		if len(o.CustomizedTargetIPList) != 0 && len(o.CustomizedTargetPortList) != 0 {
+			cmdObj = command.NewCmd(o.Protocol, o.CustomizedTargetIPList, o.CustomizedTargetPortList)
+			targetip := strings.Join(o.CustomizedTargetIPList, ",")
+			targetport := strings.Join(o.CustomizedTargetPortList, ",")
+			target = fmt.Sprintf("IPs: %s; Ports: %s", targetip, targetport)
+
```
Comment (OrangeBao) [inline pkg/command/share/do.go]: move this into ```ncat.go```

File: pkg/command/share/do.go [hunk 2]
```diff
@@ -112,35 +133,80 @@ func (o *DoOptions) RunRange(iPodInfos []*FloatInfo, jPodInfos []*FloatInfo) []*
...
+		} else if o.Protocol == string(utils.DNS) {
+			cmdObj = command.NewCmd(o.Protocol, o.TargetHostToLookup, o.TargetDNSServer)
+			var targethost string
+			var targetdns string
```
Comment (OrangeBao) [inline pkg/command/share/do.go]: move this into ```nslookup.go```

File: pkg/command/share/do.go (hunk 1 above)
Comment (GreatLazyMan) [reply pkg/command/share/do.go]: fix it

File: pkg/command/share/do.go (hunk 2 above)
Comment (GreatLazyMan) [reply pkg/command/share/do.go]: fix it

**Comment by ONE7live** [comment]
@OrangeBao Can this PR continue, or should it be merged?

**Pull request merged**